src/super_gradients/training/models/detection_models/customizable_detector.py

```diff
@@ -24,40 +24,48 @@ class CustomizableDetector(SgModule):
     Modules should follow the interface of BaseDetectionModule
     """
 
-    def __init__(self, arch_params: Union[HpmStruct, DictConfig], in_channels: int = 3):
+    def __init__(
+        self,
+        backbone: Union[str, dict, HpmStruct, DictConfig],
+        heads: Union[str, dict, HpmStruct, DictConfig],
+        neck: Optional[Union[str, dict, HpmStruct, DictConfig]] = None,
+        num_classes: int = None,
+        bn_eps: Optional[float] = None,
+        bn_momentum: Optional[float] = None,
+        inplace_act: Optional[bool] = True,
+        in_channels: int = 3,
+    ):
         """
         :param type_mapping: can be passed to resolve string type names in arch_params to actual types
```

Comment:
please fix the params in the documentation

Reply:
Added in https://github.com/Deci-AI/super-gradients/pull/760/commits/08e23d94eecbe3925473f0fcd0603ad8c63067fa